learntools/machine_learning/nbs/ch4.ipynb

```diff
@@ -78,25 +75,32 @@
     "## Step 1: Split Your Data\n",
     "Use the train_test_split command to split up your data.\n",
     "\n",
-    "Though you could use any number for the random_state, use `random_state=1` so the `check` functions know what to expect when they verify your code."
+    "Give it the argument `random_state=1` so the `check` functions know what to expect when verifying your code.\n",
```

Comment:
I don't know if we need to give them this much of an explanation - I feel like we could just as easily say "for reproducibility" and leave it at that.